packages/test/test-end-to-end-tests/src/test/counterEndToEndTests.spec.ts

```diff
@@ -219,7 +219,7 @@ describeNoCompat("SharedCounter orderSequentially", (getTestObjectProvider) => {
 
 			assert.notEqual(error, undefined, "No error");
 			assert.ok(error?.message.startsWith("RollbackError:"), "Unexpected error message");
-			assert.equal(containerRuntime.disposed, true);
+			assert.equal(container.closed, true);
```

Comment:
@clarenceli-msft FYI since you were the last to touch this. Let me know if there are any concerns about this change.

Reply:
I am fine with that